Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.9-slim
 
 WORKDIR /usr/src/app
 
-RUN apt update && apt install -y libmagic1 && rm -rf /var/lib/apt/lists/*
+RUN apt update && rm -rf /var/lib/apt/lists/*
```

Comment:
Just remove this whole line, it's not needed now.

Reply:
What you don't want to just randomly update the database and then delete it for no reason!? Don't you understand who I work for?